bundles/NewsletterBundle/README.md

```diff
@@ -0,0 +1,93 @@
+# Pimcore Newsletter Bundle
+This bundle provides a basic newsletter framework with the advantage to use all the data that is already stored in the system.
+
+### Installation
+#### Minimum Requirements
+* Pimcore >= 11
+
+#### Install
+Install bundle via composer:
+```bash 
+composer require pimcore/newsletter-bundle
+```
+Enable bundle via console or extensions manager:
```

Comment:
there's no extension manager. Bundle should be registered in `config/bundles.php`

Reply:
Sorry, I copied this from headless documents, so maybe we should update it there too.